heroku.yml

```
@@ -0,0 +1,5 @@
+build:
+  docker:
+    web: Dockerfile
+run:
+  web: cd /var/www/html/resources && bash heroku-docker-start.sh
```

Wouldn't that be enough?
```suggestion
  web: bash /var/www/html/resources/heroku-docker-start.sh
```

---

i already tried that.but showed some sort of permission denied error